### PR TITLE
Updating the README.md file for Hue

### PR DESCRIPTION
**Pull request by e55010104110**
Updating the README.md file for Hue with Hue + Hive (SQL UI) - basic SQL queries meant to guide a customer working on a POC.

Based on https://b.corp.google.com/issues/299904689#comment4

Renamed images with more descriptive file names

### CODE REVIEW

**Comment by e55010104110** [comment]
Renamed images with more descriptive file names

File: hue/README.md [hunk 1]
```diff
@@ -37,3 +37,118 @@ installed:
...
+    </property>  
+    <property>  
+        <name>oozie.service.ProxyUserService.proxyuser.\#USER\#.groups\</name\> 
+        <value>*</value>  
+    </property>
+    ```
+
+3.  Restart Oozie  
+   1.   Find the process number for the `org.apache.oozie.server.EmbeddedOozieServer  ps \-ef | grep oozie`  
+   2.   Kill the process by PID sudo kill \-9 \<PID\>
```
Comment (kuldeepkk-dev) [inline hue/README.md:150]: Stop the Oozie process with sudo systemctl stop oozie.service and if it doesn't stop successfully then you can give this option to kill it with kill -9

**Review by kuldeepkk-dev — APPROVED** [review]
LGTM

**Pull request merged**